mlxtend/classifier/stacking_cv_classification.py

```diff
@@ -138,16 +139,16 @@ def fit(self, X, y):
             for num, (train_index, test_index) in enumerate(skf):
 
                 if self.verbose > 0:
-                    print ("Training and fitting fold %d of %d..." % \
-                          ((num+1), self.n_folds))
+                    print ("Training and fitting fold %d of %d..." %
+                           ((num+1), self.n_folds))
+
+                prediction = model.fit(X[train_index], y[train_index])
```

Comment:
Here, you could just call `model.fit(X[train_index], y[train_index])` without assigning it to a parameter and use `prediction = model.predict(X[test_index])`, which would be a bit clearer in debugging